spec/Section 3 -- Type System.md

```diff
@@ -1875,6 +1875,10 @@ A GraphQL schema describes directives which are used to annotate various parts
 of a GraphQL document as an indicator that they should be evaluated differently
 by a validator, executor, or client tool such as a code generator.
 
+**Built-in Directives**
+
+:: A *built-in directive* is any directive defined within this specification.
```

Comment:
Someone can decide that directives in the examples also fall into this set (defined).

Reply:
I'm less worried about that - those are hopefully all clear that they are examples of custom directives, and beyond that there is always https://spec.graphql.org/draft/#sel-GABDLFABABrB1nR

Reply:
OK